CsvParser/csvParser_test.go

```diff
@@ -129,3 +129,26 @@ func Test_MakeJsonObjFile(t *testing.T) {
 		})
 	}
 }
+
+func Test_AddingIncidentToExistingJson(t *testing.T) {
```

Comment:
If this function is going to get added in this review, then the test for it should be finished as well.